service/matching/version_rule_helpers.go

```diff
@@ -609,17 +609,17 @@ func getSourcesForTarget(buildId string, redirectRules []*persistencespb.Redirec
 	return sources
 }
 
-// FindAssignmentBuildId finds a build ID for the given runId based on the given rules.
+// FindAssignmentBuildId finds a build ID for the given workflowId based on the given rules.
```

Comment:
Is this changing or was it used with workflow id before? Hmm, looks like it's being called with runid?

Anyway, the second line of the comment isn't consistent

Reply:
I wanted to change it, then I decided to do it later. Forgot to clean this. fixing now